Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -0,0 +1,93 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import UIKit
+
+internal final class AddNewCardViewController: UIViewController {
+  @IBOutlet fileprivate weak var cardNumberLabel: UILabel!
+  @IBOutlet fileprivate weak var cardNumberTextField: UITextField!
+  @IBOutlet fileprivate weak var cardholderNameLabel: UILabel!
+  @IBOutlet fileprivate weak var cardholderNameTextField: UITextField!
+  @IBOutlet fileprivate weak var expirationLabel: UILabel!
+  @IBOutlet fileprivate weak var expirationTextField: UITextField!
+  @IBOutlet fileprivate weak var securityCodeLabel: UILabel!
+  @IBOutlet fileprivate weak var securityCodeTextField: UITextField!
+  @IBOutlet fileprivate weak var zipCodeLabel: UILabel!
+  @IBOutlet fileprivate weak var zipCodeTextField: UITextField!
+
+  private weak var saveButtonView: LoadingBarButtonItemView!
+
+  internal static func instantiate() -> AddNewCardViewController {
+    return Storyboard.Settings.instantiate(AddNewCardViewController.self)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    let cancelButton = UIBarButtonItem(title: Strings.Cancel(),
+                                       style: .plain,
+                                       target: self,
+                                       action: #selector(cancelButtonTapped))
+    cancelButton.tintColor = .ksr_green_700
+    self.navigationItem.leftBarButtonItem = cancelButton
+
+    self.saveButtonView = LoadingBarButtonItemView.instantiate()
+    self.saveButtonView.setTitle(title: Strings.Save())
+    self.saveButtonView.addTarget(self, action: #selector(saveButtonTapped))
+    let navigationBarButton = UIBarButtonItem(customView: self.saveButtonView)
+    self.navigationItem.setRightBarButton(navigationBarButton, animated: false)
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
```

Comment:
I think here is a good opportunity to adopt keyPath instead of lenses. So, for example: 
`||> UITextField.lens.textAlignment .~ .right` would become `||> \UITextField.textAlignment .~ .right`, or the shorthand: `||> \.textAlignment .~ .right` and so on...